docs/en/sql-reference/functions/geo/h3.md

```diff
@@ -294,6 +294,52 @@ Result:
 └────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
 ```
 
+## h3PolygonToCells
```

Comment:
In fact this function should be named differently: `geoToH3Cells`

Reply:
Currently, most of your [H3 functions](https://clickhouse.com/docs/en/sql-reference/functions/geo/h3#h3togeo) match the upstream function names (with h3 prefix), which is why i went with `h3PolygonToCells`. You also already have `geoToH3` which converts a point to a h3 cell (matching the upstream h3 function).

Should I still go ahead and name it `geoToH3Cells`?